ckanext/security/authenticator.py

```diff
@@ -77,22 +77,26 @@ def authenticate(identity):
     try:
         user_name = identity['login']
     except KeyError:
+        log.debug('Login failed - missing login credentials')
         return None
 
     login_throttle_key = get_login_throttle_key(
         request, user_name)
     if login_throttle_key is None:
+        log.debug('Login failed - no login throttle key found for user %r', identity['login'])
```

Comment:
I don't think this can happen unless someone passes `''` as the login, update the log message here?

Reply:
It would happen in https://github.com/open-data/ckanext-security/blob/6ba83beffcc67d13b6ffb058a92836f2d8e3b1bf/ckanext/security/authenticator.py#L24